recipe/meta.yaml

```diff
@@ -1,16 +1,15 @@
 {% set version = "2.5.1" %}
 {% set build = 6 %}
 
-{% if cuda_compiler_version != "None" %}
-{% set build = build + 200 %}
-{% endif %}
```

Comment:
Let's keep these.

Reply:
I'm not sure if we can keep them while switching to rattler-build. I've removed these conditional macros to make conversion easier.

Reply:
I don't see a reason why we can't.